infra-templates/windows-network-services/0/README.md

```diff
@@ -0,0 +1,22 @@
+## Network Services for windows
+
+This stack provides the following services:
+
+* Metadata
+* DNS
+
+### Changelog for v0.1.0
+
+#### Metadata [rancher/rancher-metadata:v0.9.3-win]
```

Comment:
We're not planning on releasing images that have `-win`. When we make the release, I'll let you know. 

Will your changes in metadata/DNS repo affect the code used in our cattle/k8s environments?

Reply:
No, I won't change any code we use in cattle/k8s.

Reply:
As of now, I still plan on using the same tags that we use for the other code when we release these images

Reply:
So, did you mean there will be a different image name like `rancher/rancher-metadata-windows:<tag>`?